frontend/appflowy_flutter/integration_test/document/document_emoji_shortcut_test.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:appflowy_backend/protobuf/flowy-folder2/protobuf.dart';
```

Comment:
This test will not be executed in the CI job. Please add this test to `document_test_runner.dart`.